cmd/aad/aad.go

```diff
@@ -0,0 +1,234 @@
+package main
```

Comment:
Similarly to @mjudeikis's work, I would just add the necessary pieces for createorupdate in this PR and maybe have a follow-up PR to add the aad binary?

Reply:
no problem